7_react/react1/week2/todo-app/src/components/TodoItem/TodoItem.jsx

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types';
+import Task from '../Task/Task';
+import Deadline from '../Deadline/Deadline';
+import { useState } from 'react';
+import './TodoItem.css';
+function TodoItem({ todo, removeTodo }) {
+  const [isChecked, setIsChecked] = useState(false);
+
+  const handleChecboxChange = () => {
+    setIsChecked(!isChecked);
+  }
+
+  const handleRemoveClick = () => {
+    removeTodo(todo.id);
```

Comment:
Good use of callback patterns! ⭐ I would suggest going with the convention of `onSomeAction` to make it clear this is akin to listening to an event in the component.

Reply:
but React documentation has examples with `handleStuff`. Sorry, I couldn't understand what to do🙈